.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Expected Behavior
+
+
+### Actual Behavior
+
+
+### Steps to reproduce the problem
+
+
+
+### Minimal reproduction
+
+<!--
+
+This link explains why we ask for a minimal reproduction.  Thank you in advance!
+https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff
+
+You can create a reproduction here:
+https://github.com/cspotcode/ts-node-repros
```

Comment:
Do you want to move this under TypeStrong too?

Reply:
Yeah, that'd be ideal.

Reply:
@blakeembrey what's the best way to do this?  Push the code to TypeStrong/ts-node-repros?  Or use Github's "Transfer Ownership" button.